controlplane/kubeadm/internal/control_plane.go

```diff
@@ -286,3 +299,24 @@ func getKubeadmConfigs(ctx context.Context, cl client.Client, machines Filterabl
 func (c *ControlPlane) IsEtcdManaged() bool {
 	return c.KCP.Spec.KubeadmConfigSpec.ClusterConfiguration == nil || c.KCP.Spec.KubeadmConfigSpec.ClusterConfiguration.Etcd.External == nil
 }
+
+func (c *ControlPlane) PatchMachines(ctx context.Context) error {
+	errList := []error{}
+	for i := range c.Machines {
+		machine := c.Machines[i]
+		if helper, ok := c.machinesPatchHelpers[machine.Name]; ok {
+			if err := helper.Patch(ctx, machine, patch.WithOwnedConditions{Conditions: []clusterv1.ConditionType{
```

Comment:
If we keep using `WithOwnedConditions`, let's un-deprecate it

Reply:
Yep, I will open an issue and send a PR for this